### ISSUE TITLE
do we need yarn for opentsdb

### ISSUE THREAD

**Issue by vadirajks** [OpenTSDB/opentsdb#2259]
Hi Team,
I read from the documentation that opentsdb only need's hdfs and hbase.
Do i really need yarn in production setup?...if i am only setting up haddop to use opentsdb do i really need yarn?
why i need yarn if it is required?
please can anyone explain me.
Thanks and regards,
Vadiraj

**Comment by akinsse** [comment]
No, HBase can use Hadoop for columnar without Yarn.  OpenTSDB is essentially utilizing the HBase data structure and storage processes to store and recall scalar metrics. This can be done without yarn. 

If you're running any other jobs that would require a Map-Reduce process on anything in Hadoop or HBase, you'd need Yarn.
